Issue resolution updates (#391)
MicrosoftDocs/terminal: TerminalDocs/get-started.md

To customize the settings of your Windows Terminal, select **Settings** in the dropdown menu. This will open the settings UI to configure your settings. You can learn how to open the settings UI with keyboard shortcuts on the [Actions page](./customize-settings/actions.md#application-level-commands).

### Settings JSON file

If you prefer to configure your Windows Terminal settings using code, rather than the graphic user interface, you can edit the settings.json file.

Select **Settings** in the Windows Terminal dropdown menu while holding <kbd>Shift</kbd> to open the `settings.json` file in your default text editor. (The default text editor is defined in your [Windows settings](ms-settings:defaultapps).)

The path for your Windows Terminal settings.json file may be found in one of the following directories:

- Terminal (stable / general release): `%LOCALAPPDATA%\Packages\Microsoft.WindowsTerminal_8wekyb3d8bbwe\LocalState\settings.json`
- Terminal (preview release): `%LOCALAPPDATA%\Packages\Microsoft.WindowsTerminalPreview_8wekyb3d8bbwe\LocalState\settings.json`
- Terminal (unpackaged: Scoop, Chocolately, etc): `%LOCALAPPDATA%\Microsoft\Windows Terminal\settings.json`

> Settings can be customized via:

Reviewer: I'm wondering if this section is redundant with the overall docs site? It should also include overall application settings (Startup, Interaction, etc.) as well as Color schemes.

Author: Yes - I see what you mean. Removed that section. :)

> - [General profile settings](./customize-settings/profile-general.md) that are set globally in the defaults section, affecting the properties of each profile.

Reviewer: ```suggestion
- [General profile settings](./customize-settings/profile-general.md) that are specific to each profile.
```

Reviewer: The `defaults` section can contain any of the profile settings, not just the general ones.

Author: Not sure I understand this comment / update. I'm removing this section, but wondering if there is an update needed to the profile-general.md intro?

> - [Appearance profile settings](./customize-settings/profile-appearance.md) that are specific to each unique profile.

Reviewer: ```suggestion
- [Appearance profile settings](./customize-settings/profile-appearance.md) that affect the visual settings of each profile separately.
```

Author: I think I'll just remove this section based on your call-out below, but have used this suggestion to replace the intro sentence in the profile-appearance.md page.
